src/framework/ui/uitypes.h

```diff
@@ -187,9 +189,53 @@ enum class Checkable {
     Yes
 };
 
+enum class ActionCategory {
+    UNDEFINED = -1,
+    INTERNAL,
+    TABLATURE,
+    VIEWINGNAVIGATION,
+    PLAYBACK,
+    LAYOUTFORMATTING,
+    SELECTINGEDITING,
+    APPLICATION,
+    ACCESSIBILITY,
+    FILE,
+    SELECTIONNAVIGATION,
+    TEXTLYRICS,
+    CHORDSYMBOLSFIGUREDBASS,
+    MEASURES,
+    MUSICALSYMBOLS,
+    DIALOGSPANELS,
+    NOTEINPUT,
+    WORKSPACE,
+    PLUGINS
+};
+
 struct UiAction
 {
+    inline static std::vector<TranslatableString> categories
```

Comment:
I'm inclined to suggest moving this vector to ShortcutsModel, since that is the only place where it is used and I think this is rather UI-related than logic-related so might fit better into a UI-related class.

Reply:
Sure I'll move it there!